src/main/webapp/rssfeed.jsp

```diff
@@ -1,5 +1,8 @@
 <%@ page language="java" contentType="text/html; charset=ISO-8859-1"
     pageEncoding="ISO-8859-1"%>
+    
+    <!-- the super class for this needs to be created.  In specific, javax.servlet.http.HttpServlet-->

```

Comment:
Save this discussion for the pull request (where I am putting this comment; you can have inline conversations here).  It should not be in comments in the source code, because that's clutter, which is technical debt.